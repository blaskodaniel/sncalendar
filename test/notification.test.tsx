import { shallow } from 'enzyme'
import React from 'react'
import CheckCircleIcon from '@material-ui/icons/CheckCircle'
import CalendarNotification from '../src/components/notification'

describe('CalendarNotification', () => {
  const variantIcon = {
    success: CheckCircleIcon,
  }
  const testprops = {
    className: '',
    message: 'Test message',
    onClose: jest.fn(),
    variant: variantIcon.success,
  }

  it('CalendarNotification snapshot', () => {
    const wrapper = shallow(<CalendarNotification {...(testprops as any)} />)
    expect(wrapper).toMatchSnapshot()
  })
})
