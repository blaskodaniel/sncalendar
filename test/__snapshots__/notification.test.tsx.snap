// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarNotification CalendarNotification snapshot 1`] = `
<WithStyles(ForwardRef(SnackbarContent))
  action={
    Array [
      <ForwardRef(WithStyles)
        aria-label="close"
        color="inherit"
        onClick={[MockFunction]}
      >
        <Memo(CloseIcon)
          className="makeStyles-icon-5"
        />
      </ForwardRef(WithStyles)>,
    ]
  }
  aria-describedby="client-snackbar"
  className=""
  message={
    <span
      className="makeStyles-message-7"
      id="client-snackbar"
    >
      <UNDEFINED
        className="makeStyles-icon-5 makeStyles-iconVariant-6"
      />
      Test message
    </span>
  }
/>
`;
